README.rst

```diff
@@ -21,12 +26,18 @@ Optionally, you can install the library including pandas support::
 Configure
 ---------
 
-Before you start using the library, you must first set it up with your access token.
-To do that, run the following command::
+This library requires a data.world access token to work.
+Your access token can be obtained on data.world under `Settings > Advanced <https://data.world/settings/advanced>`_
+
+To configure the library, run the following command::
 
     dw configure
 
-Your API token can be obtained on data.world under `Settings > Advanced <https://data.world/settings/advanced>`_
+
+Alternatively, access tokens can be provided via the ``DW_AUTH_TOKEN`` environment variable.
+On MacOS or Unix machines, run (replacing ``<YOUR_TOKEN>>`` below with the token obtained earlier)::
+
+    export DW_AUTH_TOKEN=<YOUR_TOKEN>
```

Comment:
Should be consistent re "api token" vs "auth token".  The UI calls it an "api token".